adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/WEB-INF/tags/responsive/checkoutOrderSummary.tag

```diff
@@ -11,17 +11,17 @@
 <spring:url value="/checkout/multi/termsAndConditions" var="getTermsAndConditionsUrl"/>
 
 <%-- Components --%>
-<c:if test="${selectedPaymentMethod eq 'mbway' || selectedPaymentMethod eq 'paypal'}">
+<c:if test="${selectedPaymentMethod eq 'mbway' || selectedPaymentMethod eq 'paypal' || selectedPaymentMethod eq 'applepay'}">
     <%-- Paypal button --%>
```

Comment:
Small note: The comment and the comment below might be not fully accurate anymore.

Reply:
i will update it

Reply:
done